README.md

```diff
@@ -9,8 +9,6 @@
 
 Fury is a blazing fast multi-language serialization framework powered by **jit(just-in-time compilation)** and **zero-copy**, providing up to 170x performance and ultimate ease of use.
 
-https://furyio.org
```

Comment:
Can we reserve this? It's convenient for users to click, and our readme didn't contain link to the official website

Reply:
done